modules/amplify/predict/main.nf

```diff
@@ -0,0 +1,37 @@
+
+process AMPLIFY_PREDICT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::amplify=1.0.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/amplify:1.0.3--py36hdfd78af_0':
+        'quay.io/biocontainers/amplify:1.0.3--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(faa)
+
+    output:
```

Comment:
Possibly missing attention scores? I'm not sure exactly whether this is additional contents to the output TSV or a separte file. It might be worht checking.